tests/integration-tests/conftest.py

```diff
@@ -809,8 +809,8 @@ def extract_template(template_path):
     "us-west-2": ["usw2-az4"],
     # c5.xlarge is not supported in apse2-az3
     "ap-southeast-2": ["apse2-az1", "apse2-az2"],
-    # m6g.xlarge is not supported in apne1-az2
-    "ap-northeast-1": ["apne1-az4", "apne1-az1"],
```

Comment:
Are there any tests using `m6g.xlarge` in `ap-northeast-1`?

Reply:
Sorry, the commit message was wrong. I updated it